lib/linux_admin/chrony.rb

```diff
@@ -0,0 +1,21 @@
+module LinuxAdmin
+  class Chrony
+    def initialize(conf = "/etc/chrony.conf")
+      raise MissingConfigurationFileError, "#{conf} does not exist" unless File.exist?(conf)
+      @conf = conf
+    end
+
+    def clear_servers
+      data = File.read(@conf)
+      data.gsub!(/server\s+.+\n/, "")
```

Comment:
Is it possible to have comments or other things in this file with server in it?
Should we check that the beginning of the line starts with server?